### ISSUE TITLE
Convert the HTML documentation to markdown

### ISSUE THREAD

**Issue by rhdunn** [espeak-ng/espeak-ng#4]
The `docs` directory contains documentation in HTML format. This should be converted to markdown, with the index linked from the README.md file. The docs can be build using `kramdown`.

**Comment by ValdisVitolins** [comment]
As a start to understand language improvements better, I'll convert existing documents to markdown by changing all .html files to .md. Some questions already appear:
1. Should I use standard markdown or github flawored markdown?
2. How to comments mysterious places? As a comments in .md files or as new issues on github?

**Comment by rhdunn** [comment]
Thanks for offering to help.

1.  As a general rule, I try to make the Markdown files work in GitHub and with kramdown.
2.  I am not sure what you mean by mysterious places. If you have any questions, you can ask them here. You can create issues for any bugs you find. Any corrections and improvements you can do as part of this.

My current thoughts are:

1.  Start with a direct html to markdown conversion.
2.  Remove the extended voice documentation -- these are already incorporated into the espeak-ng project via my previous work and are documented in the README.md file.
3.  Remove references to sourceforge (this was for the espeak project), replacing them where relevant with links to the GitHub project.
4.  Perform additional cleanup and improvements.

Doing these separately will help track how the documentation has changed from the original espeak documentation.

I was thinking of using a markdown-based man page file for the command-line and C API, and generate man pages from that (as well as HTML versions). I was thinking of incorporating the Debian man pages into this as well. I can look into this (as a separate issue), if you want to focus on the language and voice file documentation.

__NOTE__: For links, I have taken the approach of referring to the .md files directly, so the links work in GitHub (see e.g. https://github.com/rhdunn/cainteoir-engine#documentation).

Thanks,
Reece

**Comment by ValdisVitolins** [comment]
Reece,
I did initial import of files and restructuring of content for index file. Other files are quite messy. If I'll be able to make another pull request without issues, you can ignore issued request and wait until I will clean up content on my forked project.

**Comment by rhdunn** [comment]
Hi,
If you make additional changes and you have selected to pull from your master branch, the pull request will automatically update so you don't need a new pull request.

**Comment by ValdisVitolins** [comment]
I've cleaned up **commands.md** file. Please take look at https://github.com/ValdisVitolins/espeak-ng/blob/master/docs/commands.md
For syntax description of html "dictionary tag" I used "quote mark" **> ...** as an indention mark. It looked OK on my test html, generated by pandoc, but github adds vertical bars for that. Is it OK?

**Comment by rhdunn** [comment]
Hi,

I have decided to take a different approach converting the documentation to markdown, as I need to figure out how it should be structured. For example, I have converted the commands.html page to a man page, based on the Ubuntu/Debian man page for espeak. This is still a work in progress.

**Comment by ValdisVitolins** [comment]
I think that is quite ok for manpage, but even better would be script which generates manpage from common documentation format.
Anyway one separate category of documentation should be for language contributors, for which markdown pages seems pretty well suited.

**Comment by rhdunn** [comment]
I have now made an initial review pass of the documentation. It is now in a state where we can improve and expand on it moving forward.

I have removed the supported languages documentation as this is continually evolving and better described within the language files. I have also removed the phoneme documentation as this is described within the referenced ASCII-IPA document by Evan Kirshenbaum. The customizations (like in the English phonemes), are better documented within the phoneme tables themselves, as those are voice specific and can and will change.
